docs/effector.md

```diff
@@ -0,0 +1,73 @@
+    @RegisterSystem
+    public class IceEffectorSystem extends BaseComponentSystem {
+        @In
+        private InWorldRenderer inWorldRenderer;
+        @ReceiveEvent
+        public void onApplyEffect(ApplyEffectEvent event, EntityRef entity, IceEffectorComponent component) {
+            EntityRef enemy = event.getTarget();
+            MovementComponent movementComponent = enemy.getComponent(MovementComponent.class);
+            double reducedSpeed = movementComponent.speed * component.slow * event.getDamageMultiplier();
+            movementComponent.speed = (float) reducedSpeed;
+            inWorldRenderer.addParticleEffect(enemy, DefenceUris.ICE_PARTICLES);
+        }
+    
+        @ReceiveEvent
+        public void onRemoveEffect(RemoveEffectEvent event, EntityRef entity, IceEffectorComponent component) {
+            EntityRef enemy = event.getTarget();
+            MovementComponent movementComponent = enemy.getComponent(MovementComponent.class);
+            movementComponent.speed = movementComponent.speed / (component.slow * event.getMultiplier());
+            inWorldRenderer.removeParticleEffect(enemy, DefenceUris.ICE_PARTICLES);
+        }
+    }
+
+### Current Implementations
+
+
+| Effector       | Effect                                                                                                                                                                                                                    | Files                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Tile                                                                                                                                                 |
+|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Are these tables likely right? I see some on the old main page that are far smaller. Another quirk of copy pasting or just my inability to picture how this would look fully rendered? 🤔

Reply:
Yes, the tables are massive in the raw Markdown because there are 4 different links embedded in them per row! They look fine in the preview though, when the links are hidden

Reply:
Bizarre! But alright 😁